templates/details/libraries/library.html

```diff
@@ -23,15 +23,18 @@ <h5>{{ item["name"]}}</h5>
       <hr class="p-separator--medium">
       <h4>Source code</h4>
     </div>
-    <pre><code>{{ library["content"] }}</code></pre>
+    <pre id="python-code"><code>{{ library["content"] }}</code></pre>
```

Comment:
I don't really have context as to why you don't want to have highlight.js try and style all possible elements so feel free to ignore this if necessary :) 

Typically you want to have highlight.js handle the highlighting on all possible elements. So in this case...

```suggestion
    <pre><code class="python">{{ library["content"] }}</code></pre>
```

Reply:
So the reason for now I scoped to only this one is because we have some code snippets that are showed in the `docstring` tab that are not python. Maybe if would be worth style all elements when we start seeing code appearing in the docstrings tab.

I made an issue for this: https://github.com/canonical-web-and-design/charmhub.io/issues/500